hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -478,9 +478,9 @@ int MDMParser::waitFinalResp(_CALLBACKPTR cb /* = NULL*/,
                     // +CREG|CGREG: <n>,<stat>[,<lac>,<ci>[,AcT[,<rac>]]] // reply to AT+CREG|AT+CGREG
                     // +CREG|CGREG: <stat>[,<lac>,<ci>[,AcT[,<rac>]]]     // URC
                     b = (int)0xFFFF; c = (int)0xFFFFFFFF; d = -1;
-                    r = sscanf(cmd, "%s %*d,%d,\"%x\",\"%x\",%d",s,&a,&b,&c,&d);
+                    r = sscanf(cmd, "%s %*d,%d,\"%x\",\"%x\",%d",s,&a,&b,&c,&d);    // XXX: Fix for buffer overrun if applicable
```

Comment:
Unclear that this needs to be fixed or looked at. Please advice. Does this look like another place to fix this and probably limit the `%s` to `%5s` or so?

Reply:
`%31s` as `char s[32]` above.

Reply:
Same below.